### ISSUE TITLE
[JS] [onnxruntime-web] WebAssembly files were not added to dist folder

### ISSUE THREAD

**Issue by juharris** [microsoft/onnxruntime-inference-examples#4]
I'm very excited to try ONNX Runtime Web. I had some issues with the quick-start_onnxruntime-web-bundler sample:
1) The sample didn't build for me because it uses `"onnxruntime-web": "^1.8.0"` which isn't released yet. So I used did `npm install onnxruntime-web@dev` and used "1.7.0-dev.4.2". The provided model.onnx worked fine with "1.7.0-dev.4.2".
2) WebAssembly files were not added to dist folder:
![image](https://user-images.githubusercontent.com/1594505/119028823-ea691700-b975-11eb-91f1-2d2b09db4cb9.png)
I was able to resolve that and another error by doing `cp node_modules/onnxruntime-web/dist/ort-wasm-threaded.wasm node_modules/onnxruntime-web/dist/ort-wasm-threaded.worker.js ./dist/` after I built.
Should I need to do that? Maybe this will be corrected in version 1.8.0?

UPDATE: I'm using `"onnxruntime-web": "^1.7.0-dev.5"` and when I build I notice:
```
WARNING in ./node_modules/onnxruntime-web/lib/wasm/binding/ort-wasm-threaded.worker.js 1:2882-2889    
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/onnxruntime-web/lib/wasm/wasm-factory.js 96:28-76
 @ ./node_modules/onnxruntime-web/lib/backend-wasm.js 11:23-53
 @ ./node_modules/onnxruntime-web/lib/index.js 18:23-48
 @ ./app.js 2:12-38
```

The provided sample worked fine with `"onnxruntime-web": ""1.7.0-dev.4.2"` even though the Wasm files weren't copied. I had some issues trying to use my model file but I'll make a separate issue about my model file.

**Comment by fs-eire** [comment]
thank you for your feedback.

for (1), yes the example document uses "1.8.0" in the package.json before we publish ORT v1.8.0. The document is prepared for v1.8 release so as it should be resolved after release. sorry for the confusing.

for (2), which exactly example do you try to run? I will try and check if I can reproduce it. if it works for `v1.7.0-dev.4.2` but failed for `v1.7.0-dev.5`, that is not expected.

**Comment by juharris** [comment]
Thank you for your quick response!

For (2), I'm using [quick-start_onnxruntime-web-bundler](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/js/quick-start_onnxruntime-web-bundler) and I saw that message when I ran `npx webpack --mode production --entry ./main.js`.

I just checked with version "1.7.0-dev.4.2" and I did NOT see the same error message but the files are also not copied to the dist/ folder.
